src/Symfony/Component/AssetMapper/MappedAsset.php

```diff
@@ -126,4 +124,9 @@ public function addJavaScriptImport(JavaScriptImport $import): void
     {
         $this->javaScriptImports[] = $import;
     }
+
+    public function getMappedContent(): ?string
```

Comment:
No need to create a getter. This is a public readonly property.

Reply:
Sorry / thanks  i originally kept content separated from mappedContent. Fixed